packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -22,12 +22,16 @@ contract SystemConfig is OwnableUpgradeable, Semver {
      * @custom:value GAS_LIMIT            Represents an update to gas limit on L2.
      * @custom:value UNSAFE_BLOCK_SIGNER  Represents an update to the signer key for unsafe
      *                                    block distrubution.
+     * @custom:value SIGNER_SET           Represents an update to the signer set.
+     * @custom:value SIGNATURE_THRESHOLD  Represents an update to the signature threshold.
      */
     enum UpdateType {
         BATCHER,
         GAS_CONFIG,
         GAS_LIMIT,
-        UNSAFE_BLOCK_SIGNER
+        UNSAFE_BLOCK_SIGNER,
+        SIGNER_SET,
```

Comment:
If the `op-node` experiences an unknown type, it will result in it erroring. The `op-node` doesn't need to know about these changes. Ideally, there are events emitted for all changes to the state of this contract so the set of events can reproduce its state but we cannot do that now so we will need to remove the 2 update types added in this PR

Reply:
Removed and split their canonical updates into new events